compiler/one-cmds/tests/README.txt

```diff
@@ -11,12 +11,12 @@ Steps:
 bin/one-prepare-venv
 ----------------------------------------------
 
-2) run 'test/prepare_test_materials.sh' to download test material models
+2) run 'tests/prepare_test_materials.sh' to download test material models
```

Comment:
Could you check this again? It is `test/` that has test materials.

Reply:
https://github.com/Samsung/ONE/blob/302627812b713393cce9b4e904862784e2e3c197/compiler/one-cmds/tests/CMakeLists.txt#L53-L66

`prepare_test_materials.sh` is installed on `test/` directory.

Reply:
Ah, I got it. `prepare_test_materials.sh` is under `tests` in source tree, but it is moved under `test` in build tree. Is it intentional? Confused. : (